scalafix-core/src/main/scala/scalafix/rewrite/ExplicitReturnTypes.scala

```diff
@@ -20,18 +23,37 @@ case class ExplicitReturnTypes(mirror: Mirror)
 
   def defnName(defn: Defn): Option[Name] = Option(defn).collect {
     case Defn.Val(_, Seq(Pat.Var.Term(name)), _, _) => name
+    case Defn.Var(_, Seq(Pat.Var.Term(name)), _, _) => name
     case Defn.Def(_, name, _, _, _, _) => name
   }
 
+  def visibility(mod: Mod): Option[MemberVisibility] = Option(mod).flatMap {
```

Comment:
You could use `collect` here and avoid the `wrapping` inside the `flatMap` as well as the fallback case.

```scala
Option(mod).collect {
  case _: Mod.Private => MemberVisibility.Private
  case _: ModProtected => MemberVisibility.Protected
}
```